basics/getting-started/frequent-questions/ingestion-faq.md

```diff
@@ -70,6 +70,13 @@ By default, Pinot limits the length of a String column to 512 bytes. If you want
     },
 ```
 
+### When does new data becomes queryable when getting ingested into real-time table?
```

Comment:
Perhaps reword as:
Events are available to be read by queries as soon as they are ingested. This is because events are instantly indexed in-memory upon ingestion. 

The ingestion of events in-memory is not a transactional event, so individual replicas of a partition are not immediately consistent. Pinot sacrifices on this consistency in favor of the availability and partition (CAP theorem) to be able to provide ultra-low read latencies at high throughput.

However, when the in-memory indexes are flushed on to persistent storage, all replicas are guaranteed to be consistent.

Reply:
```suggestion
### When does new data become queryable when getting ingested into real-time table?
```